email_notification/sender/email_notification_builder.py

```diff
@@ -146,6 +170,13 @@ def _get_by_language(self, instance, field):
             instance, f"{field}_{self.language}", getattr(instance, field, "")
         )
 
+    def _get_html_content(self, instance):
+        html_template_file = getattr(instance, f"html_content_{self.language}", "")
```

Comment:
I think `self._get_by_language` should work here

Reply:
Fixed this 👍